source/btn.cpp

```diff
@@ -33,10 +36,26 @@ static void btn_onpress(void)
     pc.printf(" done\n\r");
 }
 
+static void btn_set_fall(void)
+{
+    btn.fall(&btn_onpress);
+}
+
 void btn_init(void)
 {
     /* configure pushbutton */
     btn.mode(MAIN_BTN_PUPD);
-    wait(.01);
-    btn.fall(&btn_onpress);
+
+    /* Register the button fall handler after a short delay. */
+    minar::Scheduler::postCallback(FunctionPointer0<void>(btn_set_fall).bind())
+        .delay(minar::milliseconds(10))
+        .tolerance(minar::milliseconds(100));
```

Comment:
Technically, I don't think it matters, since this is init code where we don't really care about event coalescing 100 ms into the future.

What does matter is that this is example code. People will copy and paste this Minar-using code, so we should provide a sane default tolerance. 100 ms seems sane to me.